packages/terra-dropdown-button/src/terra-dev-site/doc/example/BlockDropdownButton.jsx

```diff
@@ -0,0 +1,25 @@
+import React, { useState } from 'react';
+
+// eslint-disable-next-line import/no-extraneous-dependencies, import/no-unresolved, import/extensions
+import DropdownButton, { Button } from 'terra-dropdown-button/lib/DropdownButton';
+
+const Example = () => {
+  const [message, setMessage] = useState('No option clicked');
+
+  return (
+    <React.Fragment>
+      <DropdownButton
+        label="Dropdown"
+        isBlock
+      >
+        <Button label="1st Option" onClick={() => setMessage('1st option clicked')} />
```

Comment:
I think it'd be better to use real-world examples (there are a couple pointed out in the [original issue](https://github.com/cerner/terra-core/issues/2216#issuecomment-501327457)) here so that it probably helps consumers better understand the difference between Dropdown Button v/s SplitButton.

Reply:
Done in f3b73577383099a9bddb5383f6a820868611af97